File: Java/books/src/main/java/es/mastercloudapps/books/model/Book.java
package es.mastercloudapps.books.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Book {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    private String title;

    private String author;

    private String publisher;

    private Integer year;

    private String summary;
}

File: Node/books/model/book.js
const mongoose = require('mongoose');

let BookSchema = new mongoose.Schema({
    title: {
        type: String,
        required: [true, 'Title is mandatory']
    },
    author: String,
    publisher: String,
    year: Number,
    summary: String
});

module.exports = mongoose.model('Book', BookSchema);